notebooks/406-3D-pose-estimation-webcam/406-3D-pose-estimation.ipynb

```diff
@@ -0,0 +1,562 @@
+    "                    engine3D.scene_remove(line_tmp)\n",
+    "                except Exception:\n",
+    "                    pass\n",
+    "\n",
+    "                engine3D.scene_add(people)\n",
+    "                line_tmp = people\n",
+    "\n",
+    "                # draw 2D\n",
+    "                draw_poses(scaled_img, poses_2d)\n",
+    "\n",
+    "            else:\n",
+    "                try:\n",
+    "                    engine3D.scene_remove(line_tmp)\n",
+    "                    line_tmp = None\n",
+    "                except Exception:\n",
+    "                    pass\n",
+    "\n",
+    "            if use_popup:\n",
+    "                frame = cv2.resize(scaled_img, dsize=(800, 600))\n",
+    "                cv2.imshow(title, frame)\n",
+    "                key = cv2.waitKey(1)\n",
+    "                # escape = 27, use ESC to except\n",
+    "                if key == 27:\n",
+    "                    break\n",
+    "            else:\n",
+    "                # encode numpy array to jpg\n",
+    "                imgbox.value = cv2.imencode(\n",
+    "                    \".jpg\",\n",
+    "                    scaled_img,\n",
+    "                    params=[cv2.IMWRITE_JPEG_QUALITY, 90],\n",
+    "                )[1].tobytes()\n",
+    "\n",
+    "            engine3D.renderer.render(engine3D.scene, engine3D.cam)\n",
+    "\n",
+    "    except KeyboardInterrupt:\n",
+    "        print(\"Interrupted\")\n",
+    "    except RuntimeError as e:\n",
+    "        print(e)\n",
+    "    finally:\n",
+    "        clear_output()\n",
+    "        if player is not None:\n",
+    "            # stop capturing\n",
+    "            player.stop()\n",
+    "        if use_popup:\n",
+    "            cv2.destroyAllWindows()\n",
+    "        if line_tmp:\n",
+    "            engine3D.scene_remove(line_tmp)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "344840a6-9660-4a11-8b05-729ac2969e28",
+   "metadata": {},
+   "source": [
+    "## Run\n",
+    "\n",
+    "### Run Live Pose Estimation\n",
+    "\n",
+    "Run using a webcam as the video input. By default, the primary webcam is set with `source=0`. If you have multiple webcams, each one will be assigned a consecutive number starting at 0. Set `flip=True` when using a front-facing camera. Some web browsers, especially Mozilla Firefox, may cause flickering. If you experience flickering, set `use_popup=True`.\n",
+    "\n",
+    "*Note: To use this notebook with a webcam, you need to run the notebook on a computer with a webcam. If you run the notebook on a server (e.g. Binder), the webcam will not work.*\n",
+    "\n",
+    "*Note: Popup mode may not work if you run this notebook on a remote computer (e.g. Binder).*"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3f82e298-5912-48c7-90b5-339aea3c177d",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# NBVAL_SKIP\n",
```

Comment:
It cannot be skipped from tests. 401 nor 402 don't skip this cell and tests pass.